src/sentry/utils/pytest/relay.py

```diff
@@ -40,8 +40,12 @@ def _remove_container_if_exists(docker_client, container_name):
             pass  # could not remove the container nothing to do about it
 
 
-@pytest.fixture(scope="session")
-def relay_server_setup(live_server, tmpdir_factory):
+@pytest.fixture(
+    scope="session",
+    params=["latest", "cc3af8314c682da977e83e86edb876bcb7da3adc"],
```

Comment:
This is so you can run relay integration tests like this:

```
pytest tests/ -krelay_v1
pytest tests/ -krelay_v2
```

and run both versions of relay, each of which will exercise the different codepaths. Running both parametrizations at once is not possible so we will need to revert this anyway (and also because this commit hash from [this PR](https://github.com/getsentry/relay/pull/778) will stop existing)